core-java-lang-oop-2/src/main/java/com/baeldung/supertypecompilerexception/MyClassSolutionI.java

```diff
@@ -0,0 +1,15 @@
+package com.baeldung.supertypecompilerexception;
+
+public class MyClassSolutionI {
```

Comment:
I think it may be clearer to use `1`, `2` etc, rather than roman numerals. `I`, `II` etc.